src/WebInterface.hs

```diff
@@ -235,6 +236,17 @@ viewPullRequest info (PullRequestId n) pullRequest =
     a ! href (toValue url) $ toHtml $ Project.title pullRequest
     span ! class_ "prId" $ toHtml $ "#" <> (show n)
 
+    case integrationStatus pullRequest of
+      Integrated _ (BuildPending (Just ciUrl)) -> do
+        span "  | "
+        a ! href (toValue ciUrl) $ "View in CI"
+
+      Integrated _ (BuildFailed (Just ciUrl)) -> do
+        span "  | "
+        a ! href (toValue ciUrl) $ "View in CI"
+
+      _ -> pure ()
```

Comment:
Awesome... and pretty handy!

Reply:
While reading the original issues I found something to improve.

Instead of just "View in CI", maybe it would be better to have: "Build pending" and "Build failed" as the link labels.